src/components/Player/Pages/Trends/Trends.jsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import { TrendGraph } from 'components/Visualizations';
+import { getPlayerMatches } from 'actions';
+import { playerMatches } from 'reducers';
+import { getCumulativeDataByField } from 'selectors';
+import { withRouter } from 'react-router';
+import FlatButton from 'material-ui/FlatButton';
+import { deSnake } from 'utility';
+import styles from './Trends.css';
+import trendNames from './trendNames';
+
+const selectTrend = (router, trendName, playerId) => {
+  router.push(`/players/${playerId}/trends/${trendName}`);
+};
+
+const Trend = ({ trendName = trendNames[0], columns, router, playerId }) => (
+  <div style={{ fontSize: 10 }}>
+    <div className={styles.buttonContainer}>
```

Comment:
This should probably be refactored into a separate "fieldSelector" component and shared between histograms and trends.  It can accept as a parameter the function to fire when a button is clicked.